packages/web/integrations/bridges/skip/queries.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  Asset,
+  Chain,
+  MsgsRequest,
+  MsgsResponse,
+  RouteRequest,
+  RouteResponse,
+  TxStatusResponse,
+} from "./types";
+
+class SkipApiClient {
+  async assets({ chainID }: { chainID?: string } = {}) {
+    const urlParams = new URLSearchParams({
+      include_evm_assets: "true",
+    });
+
+    if (chainID) {
+      urlParams.set("chain_id", chainID);
+    }
+
+    const response = await fetch(
+      `https://api.skip.money/v1/fungible/assets?${urlParams.toString()}`
+    );
+
+    const data = (await response.json()) as {
+      chain_to_assets_map: Record<string, { assets: Asset[] }>;
+    };
```

Comment:
Please take a look at our [apiClient](https://github.com/osmosis-labs/osmosis-frontend/blob/10bc2e754a7af647e5be3f5be67b8f3b73bcd5ef/packages/web/server/queries/cosmos/rpc-status.ts#L51) that will handle response types and fetch deserialization for us.